### PR TITLE
niu_mini reduce firmware size

### PR DESCRIPTION
**Pull request by XScorpion2**
## Description

Removed the audio feature from the default keymap as no audio keycodes were in it, or any source using it. The plank keymap did have the keycodes, so added the feature there.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/niu_mini/keymaps/planck/rules.mk [hunk 1]
```diff
@@ -0,0 +1 @@
+AUDIO_ENABLE = yes
```
Comment (mechmerlin) [inline keyboards/niu_mini/keymaps/planck/rules.mk]: is this set here just in case someone manages to solder a speaker in?

Comment (XScorpion2) [reply keyboards/niu_mini/keymaps/planck/rules.mk]: Ya, this keymap actually has audio keycodes in it. So it kinda made sense to keep it enabled in this case.

Comment (XScorpion2) [reply keyboards/niu_mini/keymaps/planck/rules.mk]: Actually I should remove that as the firmware is well over size.

Comment (XScorpion2) [reply keyboards/niu_mini/keymaps/planck/rules.mk]: Removed

**Pull request merged**